<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="GotoInstruct">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>30</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>961</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Controls">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 170, 255);</string>
     </property>
     <widget class="QLabel" name="Cucumber">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>430</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>530</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Dureza del vegetal</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Tama√±o de rebanada</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>140</y>
        <width>301</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="RecoilA">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>120</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">    QPushButton { 
        background-color: #304D30; 
        border: none; 
        color: #333333; 
        color: #ffffff;
        font-family: Arial, sans-serif; 
        font-size: 10px
    } 
    QPushButton:hover { 
        background-color: #163020;
        border: none;
    }

    QPushButton:pressed {
        background-color: #B6C4B6;
        border: none;
    }
    QPushButton:disabled {
        background-color: #092635;
        border: none;
        color: #999999;
    }</string>
      </property>
      <property name="text">
       <string>Retraer</string>
      </property>
     </widget>
     <widget class="QSlider" name="SliderHardness">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>500</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Regla.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>40</y>
        <width>301</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>180</number>
      </property>
      <property name="singleStep">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>281</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="RecoilB">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>250</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">    QPushButton { 
        background-color: #304D30; 
        border: none; 
        color: #333333; 
        color: #ffffff;
        font-family: Arial, sans-serif; 
        font-size: 10px
    } 
    QPushButton:hover { 
        background-color: #163020;
        border: none;
    }

    QPushButton:pressed {
        background-color: #B6C4B6;
        border: none;
    }
    QPushButton:disabled {
        background-color: #092635;
        border: none;
        color: #999999;
    }</string>
      </property>
      <property name="text">
       <string>Retraer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>180</y>
        <width>281</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Carrot">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>430</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="AdvanceA">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">    QPushButton { 
        background-color: #304D30; 
        border: none; 
        color: #333333; 
        color: #ffffff;
        font-family: Arial, sans-serif; 
        font-size: 10px
    } 
    QPushButton:hover { 
        background-color: #163020;
        border: none;
    }

    QPushButton:pressed {
        background-color: #B6C4B6;
        border: none;
    }
    QPushButton:disabled {
        background-color: #092635;
        border: none;
        color: #999999;
    }</string>
      </property>
      <property name="text">
       <string>Extender</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AdvanceB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">    QPushButton { 
        background-color: #304D30; 
        border: none; 
        color: #333333; 
        color: #ffffff;
        font-family: Arial, sans-serif; 
        font-size: 10px
    } 
    QPushButton:hover { 
        background-color: #163020;
        border: none;
    }

    QPushButton:pressed {
        background-color: #B6C4B6;
        border: none;
    }
    QPushButton:disabled {
        background-color: #092635;
        border: none;
        color: #999999;
    }</string>
      </property>
      <property name="text">
       <string>Extender</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Auto">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>120</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">    QPushButton { 
        background-color: #304D30; 
        border: none; 
        color: #333333; 
        color: #ffffff;
        font-family: Arial, sans-serif; 
        font-size: 10px
    } 
    QPushButton:hover { 
        background-color: #163020;
        border: none;
    }

    QPushButton:pressed {
        background-color: #B6C4B6;
        border: none;
    }
    QPushButton:disabled {
        background-color: #092635;
        border: none;
        color: #999999;
    }</string>
      </property>
      <property name="text">
       <string>Comenzar Secuencia </string>
      </property>
     </widget>
     <widget class="QLabel" name="Ban">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>430</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_42">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>300</y>
        <width>31</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_44">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>60</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Cortador</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_45">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>150</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Empujador</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>38</x>
        <y>10</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Cilindro de empuje</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Cilindro de corte</string>
      </property>
     </widget>
     <widget class="QPushButton" name="slicingButton">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>200</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">    QPushButton { 
        background-color: #304D30; 
        border: none; 
        color: #333333; 
        color: #ffffff;
        font-family: Arial, sans-serif; 
        font-size: 10px
    } 
    QPushButton:hover { 
        background-color: #163020;
        border: none;
    }

    QPushButton:pressed {
        background-color: #B6C4B6;
        border: none;
    }
    QPushButton:disabled {
        background-color: #092635;
        border: none;
        color: #999999;
    }</string>
      </property>
      <property name="text">
       <string>Rebanar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>390</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Advertencias:</string>
      </property>
     </widget>
     <widget class="QLabel" name="warningsLabel">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>440</y>
        <width>271</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Instructions">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 255);</string>
     </property>
     <widget class="QPushButton" name="autoFunInfo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>211</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Funcion autom√°tica</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sensorInfo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>350</y>
        <width>211</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Sensores de posicion</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Controles</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Estatus</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sliceInfo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>211</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string>Slider de rebanadas</string>
      </property>
     </widget>
     <widget class="QPushButton" name="hardInfo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>211</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Slider de dureza</string>
      </property>
     </widget>
     <widget class="QPushButton" name="advRecoInfo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>211</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string>Avance y retroceso</string>
      </property>
     </widget>
     <widget class="QLabel" name="Images">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>641</width>
        <height>421</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 18px;</string>
      </property>
      <property name="text">
       <string>Selecciona un bot√≥n para conocer informaci√≥n sobre un tema</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_43">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>390</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Uso</string>
      </property>
     </widget>
     <widget class="QPushButton" name="instructInfo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>470</y>
        <width>211</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Instrucciones de uso</string>
      </property>
     </widget>
     <widget class="QPushButton" name="protocolInfo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>430</y>
        <width>211</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Protocolo de limpieza</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rebanar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>211</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string>Funcion Rebanar</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="GotoControls">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>30</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectionButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    QPushButton { 
        background-color: #304D30; 
        border: none; 
        color: #333333; 
        color: #ffffff;
        font-family: Arial, sans-serif; 
        font-size: 10px
    } 
    QPushButton:hover { 
        background-color: #163020;
        border: none;
    }

    QPushButton:pressed {
        background-color: #B6C4B6;
        border: none;
    }
    QPushButton:disabled {
        background-color: #092635;
        border: none;
        color: #999999;
    }</string>
    </property>
    <property name="text">
     <string>Conectar</string>
    </property>
   </widget>
   <widget class="QLabel" name="connectionStateLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Desconectado</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnectionButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    QPushButton { 
        background-color: #304D30; 
        border: none; 
        color: #333333; 
        color: #ffffff;
        font-family: Arial, sans-serif; 
        font-size: 10px
    } 
    QPushButton:hover { 
        background-color: #163020;
        border: none;
    }

    QPushButton:pressed {
        background-color: #B6C4B6;
        border: none;
    }
    QPushButton:disabled {
        background-color: #092635;
        border: none;
        color: #999999;
    }</string>
    </property>
    <property name="text">
     <string>Desconectar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
